types/react-onclickoutside/index.d.ts

```diff
@@ -2,11 +2,14 @@
 // Project: https://github.com/Pomax/react-onclickoutside
 // Definitions by: Karol Janyst <https://github.com/LKay>
 //                 Boris Sergeyev <https://github.com/surgeboris>
+//                 Thomas Levy <https://github.com/NilSet>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
 import * as React from "react";
 
+export {};
```

Comment:
Why did you added empty export here?

Reply:
The lints asked me to do this if I didn't want the internal inference helper types to be automatically exported.